libraries/src/HTML/HTMLHelper.php

```diff
@@ -654,6 +654,11 @@ public static function cleanImageURL($url)
 			'height' => 0,
 		];
 
+		if ($url === null)
+		{
+			$url = '';
```

Comment:
```suggestion
			$obj->url = '';

			return $obj;
```

Maybe this would be better? Just an idea, I don't insist on it.

Reply:
Then we have duplicated code, but made it lighter which avoids the strpos

Reply:
Hmm with your latest change the url property of the returned object will not be an empty string but null. Is that intended?

Reply:
Yes sure

Reply:
Reverted it